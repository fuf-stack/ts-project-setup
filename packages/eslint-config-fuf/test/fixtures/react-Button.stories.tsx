/* eslint-disable */

// TODO: currently we only check that formatting is working at all

import type { Meta, StoryObj } from '@storybook/react-vite';

import Button from './support/Button';

const meta: Meta<typeof Button> = {
      component: Button,
};

export default meta;
type Story = StoryObj<typeof meta>;

export const Primary: Story = {
  args: {
    primary: true,
    label: 'Button',
  },
};
