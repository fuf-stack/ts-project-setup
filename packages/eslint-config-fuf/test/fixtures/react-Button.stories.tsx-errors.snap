[
  {
    "message": "Missing file extension for "./Button"",
    "ruleId": "import-x/extensions",
  },
  {
    "message": "Unable to resolve path to module './Button'.",
    "ruleId": "import-x/no-unresolved",
  },
]